README.md

```diff
@@ -0,0 +1,7 @@
+[![built with garnix](https://img.shields.io/endpoint?url=https%3A%2F%2Fgarnix.io%2Fapi%2Fbadges%2Fgarnix-io%2Fhello-garnix%3Fbranch%3Dmain)](https://garnix.io)
```

Comment:
This is currently showing “build status unknown”. Other repos show me correct results, so I’m guessing maybe this got deleted from the DB or something and that the new PR here will restore a good (but intentionally failing) build for the repo.

Reply:
I think the issue is that there previously were no branches in the DB.